Assets/XML/Text/DynamicNames/DynamicNames.xml

```diff
@@ -40,6 +40,26 @@
 	<TEXT>
 		<Tag>TXT_KEY_KINGDOM_OF</Tag>
 		<English>Kingdom of %s1</English>
+		<French>
+			<Text>Royaume du %s1</Text>
+			<Gender>Male</Gender>
+			<Plural>0</Plural>
+		</French>
+		<German>
+			<Text>K&#246;nigreich %s1:K&#246;nigreich %s1:K&#246;nigreich %s1</Text>
```

Comment:
You don't have to escape unicode characters in XML, please just write them out in plain text to make the files more legible.